automation-english/README.md

```diff
@@ -151,7 +151,7 @@ pip3 install -r requirements.txt
 
 #### Step 5: Configuring the yml config file
```

Comment:
Thanks for catching the broken link @nicorikken,
I suggest that we also update this heading to match the other one.
i.e. from `Step 5: Configuring the yml config file` to `Step 5: Configuring the yml & Updating the cover page`